_stbt/core.py

```diff
@@ -533,14 +535,51 @@ def _load_template(template):
     if isinstance(template, numpy.ndarray):
         return _AnnotatedTemplate(template, None, None)
     else:
-        template_name = _find_path(template)
-        if not template_name or not os.path.isfile(template_name):
-            raise UITestError("No such template file: %s" % template)
-        image = cv2.imread(template_name, cv2.CV_LOAD_IMAGE_COLOR)
+        relative_filename = template
+        absolute_filename = _find_user_file(relative_filename)
+        if not absolute_filename:
+            raise IOError("No such template file: %s" % relative_filename)
```

Comment:
Huh. I'm surprised because the tests passed. It seems that it falls back to the normal error message formatting if you only specify one argument:

```
>>> str(IOError("cannot identify image file 'dave.png'"))
"cannot identify image file 'dave.png'"

>>> str(IOError(errno.ENOENT, "cannot identify image file 'dave.png'", 'dave.png'))
"[Errno 2] cannot identify image file 'dave.png': 'dave.png'"
```